### PR TITLE
Updates to Core 0.8.3

### PR DESCRIPTION
**Pull request by redreceipt**
### What does this PR do, or why is it needed?

Includes new assets updates with splash screens

### How do I test this PR?

boot up app on both OS's and click around to make sure our new features work.

---

> I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))

## TODO

- [x] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [x] Closes [DEV-3717](https://newspring.atlassian.net/browse/DEV-3717)
- [ ] No new warnings in tests, in storybook, and in-app
- [ ] Upload GIF(s) of iOS and Android if applicable
- [ ] Set two relevant reviewers

## REVIEW

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

---

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

File: packages/apollos-data-prayer/package.json [hunk 1]
```diff
@@ -20,8 +20,8 @@
     "testEnvironment": "node"
   },
   "dependencies": {
-    "@apollosproject/data-schema": "^0.8.1",
-    "@apollosproject/server-core": "^0.8.1",
+    "@apollosproject/data-schema": "^0.8.3",
```
Comment (IsaacHardy) [inline packages/apollos-data-prayer/package.json:23]: Why are we skipping 0.8.2?

**Review by conrad-vanl — APPROVED** [review]
